CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif()
 set_project_warnings()
 
 # Default not to run the clang-tidy checks, default to whatever our CI_BUILD is
-option(ENABLE_CLANG_TIDY "Enable building with clang-tidy checks." ON)
+option(ENABLE_CLANG_TIDY "Enable building with clang-tidy checks." OFF)
```

Comment:
Can you fix what makes it fail the first time?

Reply:
Im not entirely sure as I am not well versed in clang-tidy but correct me if I'm wrong, I remember these things having a `warning-as-errors` option that would cause compilation to fail because clang-tidy is complaining about stuff. Maybe this is somewhere along those lines and just disabling clang-tidy will make it go away perhaps?

Reply:
nvm `warning as errors` is off by default

Reply:
@KipHamiltons  Can you elaborate, I'm not sure what you mean? Do you mean make it clang-tidy compliant?